lib/sweet_xml/options.ex

```diff
@@ -2,23 +2,23 @@ defmodule SweetXml.Options do
   @moduledoc false
 
   def handle_dtd(:all) do
-    fn _ -> [] end
+    fn _, _ -> [] end
   end
   def handle_dtd(:none) do
-    fn ets ->
-      handle_dtd(:internal_only).(ets) ++ handle_dtd(only: []).(ets)
+    fn ets, exception_module ->
+      handle_dtd(:internal_only).(ets, exception_module) ++ handle_dtd(only: []).(ets, exception_module)
     end
   end
   def handle_dtd(:internal_only) do
-    fn _ ->
+    fn _, _ ->
```

Comment:
Could you explain what's the use of this exception module ? Is it for the backward compatibility ?
Could you also make the `{:error, "no external entity allowed"}` part of the `SweetXml.DTDError`s ?

Reply:
Yes. It's for backward compatibility

> Could you also make the {:error, "no external entity allowed"} part of the SweetXml.DTDErrors ?

Done :sunglasses: